rllib/BUILD

```diff
@@ -3538,6 +3566,15 @@ py_test(
     args = ["--framework=torch", "--env=connect_four", "--win-rate-threshold=0.6", "--stop-iters=2", "--num-episodes-human-play=0"]
 )
 
+py_test(
+    name = "examples/self_play_with_open_spiel_connect_4_torch_w_rlm",
```

Comment:
"rlm" -> "rl_module"

Also: see my comment below: Can we use a command line arg here (`--use-rl-module`)?

Reply:
answered below